src/crimson/os/seastore/seastore_types.h

```diff
@@ -1593,8 +1585,9 @@ struct record_size_t {
   }
 
   void account(const delta_info_t& delta);
+
+  auto operator<=>(const record_size_t &) const = default;
```

Comment:
It doesn't make sense to implement comparison operators here.

== operator is enough.

Reply:
@cyx1231st fixed and repushed